addons/mail/static/src/messaging/component/chatter/chatter.js

```diff
@@ -77,11 +77,25 @@ class Chatter extends Component {
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     */
+    _onAttachmentsChanged() {
+        if (this.chatter.hasRecordReloadOnAttachmentCreated) {
+            this.trigger('o-reload');
+        }
+    }
+
     /**
      * @private
      */
     _onComposerMessagePosted() {
         this.chatter.update({ isComposerVisible: false });
+        // FIXME see with suggestedrecipients, we could only trigger if posted
+        // message contains partners
+        if (this.chatter.hasRecordReloadOnMessagePosted) {
+            this.trigger('o-reload');
+        }
```

Comment:
I would just always trigger the event and let the parent handle it if he wants to. Then the prop can be removed?